scripts/lldbinit.py

```diff
@@ -55,8 +55,8 @@ def print_usage() -> None:
        the `Label` string in `IdBase` classes to find possible TYPE names,
        though only Id types that have a matching `Make...Id()` function are
        supported.
-  ID is an integer number, such as `42`. This can be in hex (without the typical
-       0x prefix), such as `6000A`.
+  ID is an integer number, such as `42`, in hex (without the typical 0x prefix),
+     such as `6000A`.
```

Comment:
Maybe give a specific example? Then you could also drop the "typical 0x prefix" if you wanted.

```suggestion
     such as `inst6000000A`.
```